packages/nimbus/src/components/button/button.mdx

```diff
@@ -17,153 +17,767 @@ figmaLink: >-
-the Button.
+Use this button when the user is approaching a permanent destructive action.
 
 ```jsx-live
-const App = () => {
-  const variants = ["solid", "outline", "ghost", "link"];
-  const colors = ["primary", "critical"];
+const App = () => (
+  <Button tone="critical" size="md" variant="solid">
+    <Icons.SentimentSatisfied />
+      Button
+    <Icons.SentimentSatisfied />
+  </Button>
+)
+```
 
-  return (
-    <Stack>
-      {colors.map((color) => (
-        <Stack key={color} direction="horizontal">
-          {variants.map((variant) => (
-            <Button tone={color} key={variant} variant={variant}>
-              '{variant}' Button
-            </Button>
-          ))}
-        </Stack>
-      ))}
-    </Stack>
-  );
-};
+## Guidelines
+
+Buttons are essential interactive elements that empower users to trigger actions
+and navigate within your product. They serve as clear calls to action, guiding
+users through key flows and enabling them to interact with content in meaningful
+ways. These guidelines ensure consistent styling and behavior for all buttons,
+contributing to a cohesive and predictable user experience.
+
+### Writing guidelines
+
+- A good button label tells you exactly what will happen when you click it,
+  making the whole experience more intuitive and enjoyable.
+- Use concise and clear action based language for all buttons.
+- Use sentence case, capitalize the first word and use lowercase for the rest.
+- Be aware that with localization/translation character count will add on
+  average 20% more length.
+
+### Preferred words
+
+| **Text**                                | **Description**                                                                                                                                                                                                                                                                                              |
+| --------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| **Save**                                | Carries out pending changes, commits to saving the information.                                                                                                                                                                                                                                              |
+| **Back**                                | This term typically implies a return to a previous state or location in a non-linear or less structured flow. It's often used for:<ul style={{listStyle: "disc"}}><li>Navigating browser history</li><li>Exiting a screen or section</li></ul>                                                               |
```

Comment:
```suggestion
| **Back**                                | This term typically implies a return to a previous state or location in a non-linear or less structured flow. It's often used for:<ul style={{listStyle: "disc", marginLeft: "20px"}}><li>Navigating browser history</li><li>Exiting a screen or section</li></ul>                                                               |
```

Reply:
Such a simple trick, why didn't I think of that! Thank you.